wlms/ircbridge.go

```diff
@@ -2,12 +2,12 @@ package main
 
 import (
 	"log"
-
 	"github.com/thoj/go-ircevent"
+	"strings"
 )
 
 type IRCBridger interface {
-	Connect(chan Message, chan Message) bool
+	Connect(chan Message, chan Message, chan string, chan string) bool
```

Comment:
add documentation on the parameters. They become unwieldy.

Also, can you instead pass in a struct that has named members?